lib/ui/widgets/menuButton/downloadButton.dart

```diff
@@ -93,16 +94,12 @@ class _DownloadButtonState extends State<DownloadButton> {
   }
 
   void onDownload() async {
-    var status = await Permission.storage.status;
-    if (!status.isGranted) {
-      await Permission.storage.request();
-    }
     setState(() {
       isLoading = true;
     });
     print(widget.link);
     toasts.codeSend("Starting Download");
-    GallerySaver.saveImage(widget.link, albumName: "Prism").then((value) {
+    await ImageDownloader.downloadImage(widget.link).then((value) {
```

Comment:
Also by using the Image Downloader package, the downloaded images won't be saved in a Directory named Prism which was earlier possible with GalllerySaver package. Could you fix this as well?

Reply:
It'll will be saved in both gallery and the app directory too. @LiquidatorCoder

Reply:
Actually, I want to move the downloaded image to a new folder named Prism `Pictures\Prism` . So that our users will download all the wallpapers inside this directory for convenience,  which is also visible in the Gallery.

Reply:
Can you merge the notch display feature for this part? I will work on the directory feature. @LiquidatorCoder

Reply:
Okay I will merge that.